rpaas/api.py

```diff
@@ -232,6 +232,46 @@ def list_routes(name):
         return "Instance not found", 404
 
 
+@api.route("/resource/<name>/block", methods=["POST"])
+@auth.required
+def add_block(name):
+    content = request.form.get('content')
+    block_name = request.form.get('block_name')
+    if block_name not in ('server', 'http'):
+        return 'please, send block_name for this block (server or http)', 400
+    if not content:
+        return 'please, send content for this block', 400
+    try:
+        get_manager().add_block(name, block_name, content)
+    except manager.NotReadyError as e:
+        return "Instance not ready: {}".format(e), 412
+    return "", 201
+
+
+@api.route("/resources/<name>/block", methods=["DELETE"])
```

Comment:
What do you thing about having the route as "DELETE /resources/<name>/block/<block_name>". This would prevent questions about whether block_name should be sent as a query string or in the body. This is an ongoing argument when the DELETE method is used as the HTTP spec isn't clear about it and everyone has an opinion.